homeassistant/components/sensor/influxdb.py

```diff
@@ -173,9 +173,11 @@ def update(self):
 
         points = list(self.influx.query(self.query).get_points())
         if len(points) == 0:
-            _LOGGER.error('Query returned no points : %s', self.query)
-            return
-        if len(points) > 1:
-            _LOGGER.warning('Query returned multiple points, only first one'
-                            ' shown : %s', self.query)
-        self.value = points[0].get('value')
+            _LOGGER.info('Query returned no points, sensor state set'
```

Comment:
For consistency, keep at least the same level as below `warning` or leave it as `error`. Otherwise it might just be lost in the logs